docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_vision_feature_label_correlation:
+
+Property Label Correlation
+**************************
+
+This notebook provides an overview for using and understanding the "Property Label Correlation" check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+=================================
+The check estimates for every image property (such as brightness, contrast etc.)
+its ability to predict the label by itself.
+
+This check can help find a potential bias in the dataset, that might lead to shortcut learning - the labels being
+strongly correlated with simple image properties such as color, brightness, aspect ratio and more. This is a critical
+problem, because the model is likely to learn this property instead of the actual visual characteristics of each class,
+as it's easier to do so. In this case, the model will show high performance on images taken in similar conditions, but
+will fail in the wild, where the simple properties don't hold true.
+This kind of correlation will likely stay hidden without this check until tested in the wild.
+
+A famous example is the case of wolves vs. dogs classification, where a model needs to classify whether an image
+contains a wolf or a dog, and can learn to do it by the background instead of the actual animal - in the dataset
+most of the wolves were photographed in the snow and therefore had a white background while all the dogs were
+photographed in the grass and therefore had a green background.
+
+The check is based on calculating the predictive power score (PPS) of each image
+property. For more details you can read here `how the PPS is calculated
+<#how-is-the-predictive-power-score-pps-calculated>`__.
+
+What is a problematic result?
+-----------------------------
+
+Image properties with a high predictive score can indicate that there is a bias in the dataset, as a single property
+can predict the label successfully using simple classic ML algorithms.
+
+This means that a deep learning algorithm may accidentally learn these properties instead of more accurate complex
+abstractions. For example, in the dataset of wolves and dogs photographs, the brightness of the image may be used to
+predict the label "wolf" easily.
+
+
+How do we calculate for different vision tasks?
```

Comment:
I suggest changing this to
"How do we calculate the predictive power for different vision tasks"
or "How do we calculate the correlation between the properties and the label for different vision tasks"

(mainly to improve the sentence's structure / grammar)